types/react/test/index.ts

```diff
@@ -402,7 +402,13 @@ const ForwardingRefComponent = React.forwardRef((props: {}, ref: React.Ref<RefCo
     return React.createElement(RefComponent, { ref });
 });
 
-const ForwardingRefComponentPropTypes: React.WeakValidationMap<Props> = {};
+interface AttributeProps extends React.Attributes {
```

Comment:
```suggestion
interface AttributeProps {
```

`React.Attributes` is not meant to be used that way since it includes props that you never see inside your component or need to validate with `propTypes`. It's technically only a private type.

Reply:
The test is broken without `extends React.Attributes` -- `WeakValidationMap<AttributeProps>` doesn't have any properties in common with `WeakValidationMap<RefAttributes<RefComponent>>` from `ForwardRefExoticComponent<RefAttributes<RefComponent>>`

Does that mean the test is broken or the new propTypes property in ForwardRefExoticComponent is wrong? I suppose I could say

```ts
const ForwardingRefComponentPropTypes: React.WeakValidationMap<React.RefAttributes<RefComponent>> = {};
```

in the test instead.

Reply:
I don't know React that well, I'm just trying to get it unblocked for other contributors, so any advice you have is helpful.

Reply:
Well lets merge this if this block other work. I'll take  look later why we need this.

Reply:
Thanks. I merged it.

@rpokorny Maybe you can take another look too?